### PR TITLE
UI: Allow choosing bgs (android)

### PR DESCRIPTION
**Pull request by xiushudongfang**
It is only for android.

### CODE REVIEW

**Comment by LunaMoo** [comment]
Sorry Unknown(miss click:P), this looks fine.;3

File: UI/GameSettingsScreen.cpp [hunk 1]
```diff
@@ -950,6 +950,10 @@ UI::EventReturn GameSettingsScreen::OnChangeBackground(UI::EventParams &e) {
 		if (System_GetPropertyInt(SYSPROP_HAS_IMAGE_BROWSER)) {
 			System_SendMessage("bgImage_browse", "");
 		}
+#if defined(__ANDROID__)
+        System_SendMessage("bgImage_select",
```
Comment (hrydgard) [inline UI/GameSettingsScreen.cpp]: wonky indentation, use tabs.

**Review by hrydgard — COMMENTED** [review]
Looks good to me, small code style issue.

File: UI/GameSettingsScreen.cpp (hunk 1 above)
Comment (unknownbrackets) [inline UI/GameSettingsScreen.cpp]: Why can't both platforms just use bgImage_browse?  If it needs the path parameter, it won't hurt for browse to take it.

-[Unknown]

File: android/src/org/ppsspp/ppsspp/NativeActivity.java [hunk 2]
```diff
@@ -911,6 +918,68 @@ public boolean onKeyUp(int keyCode, KeyEvent event) {
...
+			Cursor cursor = getContentResolver().query(selectedImage,
+					filePathColumn, null, null, null);
+			cursor.moveToFirst();
+
+			int columnIndex = cursor.getColumnIndex(filePathColumn[0]);
+			String picturePath = cursor.getString(columnIndex);
+			cursor.close();
+			String imgName = "";
+			imgName =  picturePath.endsWith(".jpg")? "background.jpg":"background.png";
+			System.out.println(picturePath);
```
Comment (unknownbrackets) [inline android/src/org/ppsspp/ppsspp/NativeActivity.java]: Wouldn't it be better to remove the debugging lines now?

Also, it'd probably be cleaner if this...

```java
String imgName = "";
imgName =  picturePath.endsWith(".jpg")? "background.jpg":"background.png";
```

Was just:

```java
String imgName = picturePath.endsWith(".jpg") ? "background.jpg" : "background.png";
```

No need to set it to an empty string first when you're going to immediately set it to something else, right?

That said, it seems like copying files in Java is painful.  I wonder if it'd be better to send `bgImage_updated` the `picturePath` string, and then have the bgImage_updated code do...

```c++
		if (!value.empty()) {
			std::string dest = GetSysDirectory(DIRECTORY_SYSTEM) + (endsWithNoCase(value, ".jpg") ? "background.jpg" : "background.png");
			File::Copy(value, dest);
		}
		UIBackgroundShutdown();
		UIBackgroundInit(*uiContext);
```

That seems like it'd be simpler, and avoid the need for this slow looking copy function in Java.

-[Unknown]

**Comment by xiushudongfang** [comment]
Oh,I mess my branch. I am fixing.....

**Pull request closed without merging**